index.html

```diff
@@ -32,11 +32,19 @@ <h2>Experience</h2>
             industry.</p>
     </section>
     <section>
+        <section id="skills">
+            <h2>skills</h2>
+            <ul></ul>
+        </section>
         <h2>Connect</h2>
         <ul>
             <li><a href="https://github.com/Haregebezu" target="_blank"> GitHub </a></li>
             <li><a href="https://www.linkedin.com/in/haregebezu-amare/" target="_blank"> Linkedin </a></li>
         </ul>
     </section>
+    <footer>
+
+    </footer>
+    <script src="./JavaScripit/index.js"></script>
```

Comment:
```suggestion
    <script src="./JavaScript/index.js"></script>
```
It will work as long as you're consistent, but typos can be confusing down the line. I'd also suggest going with a plain lowercase directory name. Unless something requires mixed case, keeping it all lower means it's harder to mess up.